classes/patterns/class-highlightedcta.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Highlighted CTA pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Highlighted CTA.
+ *
+ * @package P4GBKS\Patterns
+ */
+class HighlightedCta extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/highlighted-cta';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		return [
+			'title'      => __( 'Highlighted CTA', 'planet4-blocks-backend' ),
+			'categories' => [ 'planet4' ],
+			'content'    => '
+				<!-- wp:columns {"className":"block","textColor":"white","backgroundColor":"dark-green"} -->
+					<div class="wp-block-columns block has-dark-green-background-color has-text-color has-background has-white-color">
+						<!-- wp:column -->
+							<div class="wp-block-column">
+								<!-- wp:image {"align":"center"} -->
+									<div class="wp-block-image">
+										<figure class="aligncenter">
+											<img alt=""/>
+										</figure>
+									</div>
+								<!-- /wp:image -->
+								<!-- wp:heading {"placeholder":"Enter text","align":"center","className":"has-text-align-center","level":3} -->
```

Comment:
Can we translate this string.

Reply:
Good catch! [Fixed](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/814/commits/d9ec3d7ac1c22e91dd7cdf39432ec4417af4f8f6)

Reply:
I've actually updated the [commit](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/814/commits/93a42c78f80a2784c8cd07a30026875bc89f79db) in order to add translations to the first pattern that we created, the Side image with CTA and text, because we forgot them there 😬 I've also removed the placeholder from the Button there, because this block actually doesn't have a placeholder attribute at the moment.